src/main/java/com/google/devtools/build/lib/runtime/CommonCommandOptions.java

```diff
@@ -56,6 +56,19 @@
               + "your build may break in the future due to deprecations or other changes.")
   public Void allIncompatibleChanges;
 
+  @Option(
+      name = "enable_platform_specific_config",
+      defaultValue = "false",
+      documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "When the value of this flag is true, a host platform specific config section will be "
```

Comment:
> When the value of this flag is true

Could be rewritten as "If true".

> ... a host platform specific config section will be enabled if it exists

I find that confusing. Using active voice and straight word order feels clearer to me. How about: "... Bazel picks up host-OS-specific config lines from the bazelrc files."

I feel like an example would greatly help too: "For example, if the host OS is Linux and you run bazel build, Bazel would pick up lines starting with build:linux"

> linux, osx, windows

macOS is the official name so let's use "macos" instead of "osx".

> It's equivalent to add --config=linux on Linux platforms and --config=windows on Windows, etc.

We could be a little clearer here: "Enabling this flag is equivalent to using --config=linux on Linux, --config=windows on Windows, etc."

Putting it together:

> If true,  Bazel picks up host-OS-specific config lines from bazelrc files. For example, if the host OS is Linux and you run bazel build, Bazel picks up lines starting with build:linux. Supported OS identifiers are linux, macos, windows, and freebsd. Enabling this flag is equivalent to using --config=linux on Linux, --config=windows on Windows, etc.

Reply:
Wow, thanks for helping with the documentation. This is great improvement!

As for the name for mac, it seems we still use `osx` in a lot of places in our code base. For example, https://github.com/bazelbuild/bazel/blob/71a213c42a4572f64b323b3db731e1334b6a0c38/tools/platforms/BUILD#L63

It prefer to keep the consistence util we decided to migrate everything to `macos`. WDYT?

Reply:
Could you please ask the Apple team what's their preference?